lib/Common/Memory/Recycler.h

```diff
@@ -2353,6 +2385,7 @@ class ListTypeAllocatorFunc<Recycler, isLeaf>
 // Dummy class to choose the allocation function
 class RecyclerLeafAllocator;
 class RecyclerNonLeafAllocator;
+class RecyclerWriteBarrierAllocator;
```

Comment:
because not all NonLeaf are allocated with software write barrier, on windows, swb is only enabled in a few class

---
In reply to: [86644009](https://github.com/Microsoft/ChakraCore/pull/1920#discussion_r86644009) [](ancestors = 86644009)